examples/fscloud/variables.tf

```diff
@@ -6,28 +6,24 @@ variable "ibmcloud_api_key" {
 
 variable "prefix" {
   type        = string
-  default     = "test-cos-fscloud"
   description = "Prefix name for all related resources"
 }
 
 variable "resource_tags" {
   type        = list(string)
   description = "Optional list of tags to be added to created resources"
-  default     = []
```

Comment:
`resource_tags` is optional, so put back the default value of `[]`, otherwise it becomes a required variable.